src/yape06/yape06.h

```diff
@@ -52,17 +52,14 @@ class Yape06 {
     while (--x >= 0) {
```

Comment:
i don't think this is necessary. The Array is allocated with zeros values, probably we can remove this code.